lib/src/main/java/graphql/nadel/engine/blueprint/NadelExecutionBlueprint.kt

```diff
@@ -16,7 +16,7 @@ import graphql.schema.GraphQLSchema
  */
 data class NadelOverallExecutionBlueprint(
     val engineSchema: GraphQLSchema,
-    val fieldInstructions: Map<FieldCoordinates, List<NadelFieldInstruction>>,
+    val fieldInstructions: NadelFieldMap<List<NadelFieldInstruction>>,
```

Comment:
This is a new construct that lets us query by `get(typeName: String, fieldName: String)` so we don't have to construct a `FieldCoordinates` just to perform a lookup.